backend/model.json

```diff
@@ -1,11 +1,23 @@
 {
   "weights": [
-    0.61640950087415, 0.005685696214227982, 0.0007652819723800386,
-    0.008120856648274982, 0.006471121878617543, 0.002290149111894504,
-    0.6122148431174362, 0.009226268475262742, 0.3097209600218586,
-    -1.68001441476926, -0.1574291497177478, 0.5688171086937409,
-    -0.7373752959175748, 2.9067139758034513, -0.8188878159169419,
-    -0.46518195852380023, -0.08373708520819, -0.5702255435746328
+    -0.2627561717307637,
+    0.004036564012990808,
+    -0.2650314818025725,
+    0.007231030588378769,
+    0.0001514559895926926,
+    0.0046255514612040935,
+    0.004316814071644439,
+    0.009412997849606631,
```

Comment:
I noticed the weights keep changing in every PR, are you re-training the model?

Reply:
Yes, but I'm going to have to update it to only train at intervals instead of anytime I fetch recommendations